compute/ip_address_associations_test.go

```diff
@@ -0,0 +1,163 @@
+package compute
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/go-oracle-terraform/helper"
+	"github.com/hashicorp/go-oracle-terraform/opc"
+)
+
+const (
+	_IPAddressAssociationTestName        = "test-acc-ip-address-association"
+	_IPAddressAssociationTestDescription = "testing ip address association"
+)
+
+func TestAccIPAddressAssociationsLifeCycle(t *testing.T) {
+	helper.Test(t, helper.TestCase{})
+
+	iClient, nClient, vnClient, err := getVirtNICsTestClients()
```

Comment:
Should create a new function here `getIPAddressAssociationsClients()` that return all the required clients for this test.